live-examples/js-examples/intl/intl-locale.js

```diff
@@ -1,7 +1,7 @@
 const korean = new Intl.Locale('ko', {
   script: 'Kore',
   region: 'KR',
-  hourCycle: 'h24',
+  hourCycle: 'h12',
```

Comment:
Maybe use `h23`? Otherwise we have two `h12`s.

Reply:
Done.